Clarify need for SNI (#161)
MikeBishop/dns-alt-svc: draft-ietf-dnsop-svcb-httpssvc.md

This specification does not alter the DNS queries performed when connecting
to an Alt-Svc hostname (typically A and/or AAAA only).

## Requiring Server Name Indication

Clients MUST NOT use an HTTPSSVC response unless the
client supports TLS Server Name Indication (SNI) and 
indicate the origin name when negotiating TLS.
This supports the conservation of IP addresses.

> Note that the SNI information provided in TLS by the client will be
> that of the origin, not the SvcDomainName (as will the Host HTTP header
> field value).

Reviewer: ```suggestion
Note that the TLS SNI (and also the HTTP "Host" or ":authority") will indicate 
the origin, not the SvcDomainName.
```
